core/arch/arm/plat-imx/config/config_imx7.h

```diff
@@ -12,8 +12,3 @@
 #ifndef CFG_DDR_SIZE
```

Comment:
i think you can drop all those CFG_DDR_SIZE check and consolidate it in the platform_cofig.h if really needed

Reply:
ok